samples/sensor/bme280/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _bme280:
+
+BME280 Humidity and Pressure Sensor
+###################################
+
+Overview
+********
+
+This sample application periodically reads temperature, pressure and humidity data from
+the first available device that implements SENSOR_CHAN_AMBIENT_TEMP, SENSOR_CHAN_PRESS
+and SENSOR_CHAN_HUMIDITY. This sample checks the sensor in polling mode (without
+interrupt trigger).
+
+Building and Running
+********************
+
+This sample application uses an BME280 sensor connected to an Arduino 101 board via I2C.
+Sensor has multiple pins so you need to connect according to connection diagram given in
+`bme280 datasheet`_ at page 38.
+There are two processor cores (x86 and ARC) on the Arduino 101. You will need to
+flash both this sample's code on the ARC core (using the ``arduino101_ss`` board target),
+and stub code on the x86 core (using the ``arduino_101`` board target), as shown below.
+
+.. code-block:: console
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensors/bme280
+   :board: arduino_101_sss
+   :goals: flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: tests/booting/stub
+   :board: arduino_101
+   :goals: flash
+   :compact:
+
+Sample Output
+=============
+To check output of this sample , any serial console program can be used.
+Here I am using picocom program to open output. Check which tty device it is.
```

Comment:
Change to:
```
This example uses ``picocom`` on the serial port ``/dev/ttyUSB0``:
```

Reply:
@dbkinder Thank you for those suggestions. Should I need to change this file locally in my PC, and check it manually? Is there any way that accepts your commits and pull it down to my PC?

Reply:
It seems to work better to make the change locally and do a "git commit --amend" and a forced push to update your PR.

Reply:
Done. Please help me to review again. Thanks.

Reply:
@dbkinder

Reply:
It doesn't appear that my previous comments were addressed yet?